changes/ce/feat-10391.en.md

```diff
@@ -1 +1,7 @@
-hide exhook/rewrite/topic_metric/persistent_session_store/overload_protection from the docs and configuration file.
+Hide a large number of advanced options to simplify the configuration file.
+
+That includes `exhook`, `rewrite`, `topic_metric`, `persistent_session_store`, `overload_protection`,
```

Comment:
Please change exhook, rule_engin, bridge to `low` importance level instead of hidden.